### ISSUE TITLE
Unexpected token / in JSON at position 0

### ISSUE THREAD

**Issue by bikas-kumar** [maciejtreder/ng-toolkit#815]
Using Angular cli version 10.
Node version 12.x
![errr](https://user-images.githubusercontent.com/40659871/87804108-0c36d080-c871-11ea-92fb-ce86c9740204.PNG)

**Comment by bikas-kumar** [comment]
Is ng-toolkit latest version compatible with Angular 10??

**Comment by bikas-kumar** [comment]
Angular dependencies.
![image](https://user-images.githubusercontent.com/40659871/87804471-8d8e6300-c871-11ea-96e8-fb2024abeec5.png)

**Comment by KingDarBoja** [comment]
Nope, it is not compatible with Angular 10 at the moment. I suggest using nx-plugins as explained on [this issue](https://github.com/flowaccount/nx-plugins/issues/25) if you need serverless + universal + Angular 10 as soon as possible,

**Comment by bikas-kumar** [comment]
Which latest Angular version does it supports?

**Comment by KingDarBoja** [comment]
> Which latest Angular version it supports?

Angular v8 unfortunately.

**Comment by KingDarBoja** [comment]
Working on it at #826
